packages/fxa-content-server/app/scripts/templates/sms_send.mustache

```diff
@@ -3,7 +3,15 @@
       <div class="error"></div>
     </header>
     <section class="send-sms">
-      <div class="graphic {{graphicId}}">{{#t}}Success{{/t}}</div>
+      <div class="graphic {{graphicId}}" role="img" aria-label="{{#t}}Connect another device{{/t}}"></div>
+      <h1>
+        {{^userHasAttachedMobileDevice}}
+          {{#t}}Would you like to sync your phone?{{/t}}
```

Comment:
I know this is in the SMS doc, so feel free to disregard, but I wonder if this should be "device" instead of "phone", considering other verbiage uses "devices" (and after all, you can send to your tablet).

Reply:
I think they _want_ that differentiation here. If the user doesn't have a `deviceType` of `mobile` on their account, we want to ask if they'd like to sync their phone. If they do have a phone attached, we ask if they want to keep syncing devices.

Reply:
Got it! Thanks for the clarification.